pkgs/tools/networking/edgedb/default.nix

```diff
@@ -19,24 +19,28 @@
 }:
 rustPlatform.buildRustPackage rec {
   pname = "edgedb";
-  version = "5.4.1";
+  version = "5.5.2";
 
   src = fetchFromGitHub {
     owner = "edgedb";
     repo = "edgedb-cli";
     rev = "refs/tags/v${version}";
-    hash = "sha256-qythVPcNijYmdH/IvyoYZIB8WfYiB8ByYLz+VuWGRAM=";
+    hash = "sha256-CSs1Ql0zsGgSmZrlZIfj2pJdtAax7HUlfCq8oTbReng=";
     fetchSubmodules = true;
   };
 
+  # When upgrading, please include a line with a link to the diff
```

Comment:
I would remove this, as this is not edgedb specific requirement but nixpkgs 

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#commit-conventions

Reply:
Interesting, this is the first time I see such a requirement enforced, I even thought it applied to PR messages, not to commits. I'll remove the extra node then.

Reply:
yes, this is frequently brought up, nixpkgs is huge project and everyone is on different levels of knowledge,

but you can see that is frequently applied 

```
git log | grep Diff
```